src/libs/actions/Card.ts

```diff
@@ -499,6 +500,46 @@ function updateExpensifyCardLimitType(workspaceAccountID: number, cardID: number
     API.write(WRITE_COMMANDS.UPDATE_EXPENSIFY_CARD_LIMIT_TYPE, parameters, {optimisticData, successData, failureData});
 }
 
+function deactivateCard(workspaceAccountID: number, cardID: number, oldCardState?: ValueOf<typeof CONST.EXPENSIFY_CARD.STATE>) {
+    const authToken = NetworkStore.getAuthToken();
+
+    if (!authToken) {
+        return;
+    }
+
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${CONST.EXPENSIFY_CARD.BANK}`,
+            value: {
+                [cardID]: {
+                    state: CONST.EXPENSIFY_CARD.STATE.STATE_DEACTIVATED,
+                },
```

Comment:
We will remove the card now that is not offline supported
```suggestion
                [cardID]: null
```